docs/source/guide/testcase.rst

```diff
@@ -230,7 +154,7 @@ Adding a tag via Test Case view
 Removing a tag via Test Case view
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-#. Open the Test Case to be edited, click the **Tags** tab.
+#. Open the Test Case to be edited, navigate to **Tags** card and input a tag.
```

Comment:
This is for `Adding a tag via Test Case view`. Removing is just clicking the red X icon. IMO we can just remove these sections b/c adding/removing of tags is more natural now and doesn't need a special mention in the documentation.

Also see the above section for adding tags and its screenshot image.
Both |The Test Case tags tab 1| and |The Test Case tags tab 2| images no longer match reality.